asset_dashboard/models.py

```diff
@@ -248,3 +248,205 @@ class DummyProject(models.Model):
     def __str__(self):
         """"String for representing the model object"""
         return self.name
+
+
+class NaturePreserves(models.Model):
+
+    class Meta:
+        managed = False
+        db_table = '"quercus"."nature_preserves"'
+
+    id = models.AutoField(primary_key=True, db_column='nature_preserves_id')
```

Comment:
When a primary key isn't explicitly specified, Django tries to use `id`. There isn't an `id` column in many of these tables, so you have to specify it in `db_column`.